services/rfq/relayer/service/handlers.go

```diff
@@ -201,20 +233,37 @@ func (q *QuoteRequestHandler) handleCommitPending(ctx context.Context, span trac
 		return nil
 	}
 
-	bs, err := q.Origin.Bridge.BridgeStatuses(&bind.CallOpts{Context: ctx}, request.TransactionID)
+	var bs uint8
+	call := func(ctx context.Context) error {
```

Comment:
If we're going to be doing this all over the place, I'd recommend instead wrapping [ethergo/client](https://pkg.go.dev/github.com/synapsecns/sanguine/ethergo/client#DialBackend) and having a func called something like `DialRetryableBackend`

Alchemy has a good [guide](https://docs.alchemy.com/docs/how-to-implement-retries) on retries that might be good supplemental reading, but exponential retries w/ cap seems like the way to go.

I say this because there's nothing "special" about this call, these can all fail and we want to retry on *http errors*, not evm retry errors